app/src/main/java/io/github/mattpvaughn/chronicle/data/local/SharedPreferencesPrefsRepo.kt

```diff
@@ -191,6 +201,16 @@ class SharedPreferencesPrefsRepo @Inject constructor(private val sharedPreferenc
         get() = sharedPreferences.getLong(KEY_REFRESH_RATE, defaultRefreshRate)
         set(value) = sharedPreferences.edit().putLong(KEY_REFRESH_RATE, value).apply()
 
+    private val defaultJumpForwardSeconds = 30L
+    override var jumpForwardSeconds: Long
+        get() = sharedPreferences.getLong(KEY_JUMP_FORWARD_SECONDS, defaultJumpForwardSeconds)
+        set(value) = sharedPreferences.edit().putLong(KEY_JUMP_FORWARD_SECONDS, value).apply()
+
+    private val defaultJumpBackwardSeconds = 30L
```

Comment:
could this be the original default backwards skip of 10s?

Reply:
Makes sense